src/containers/sign-on/store/sagas.js

```diff
@@ -484,6 +524,17 @@ function* watchConfigureMetaMask() {
   yield takeLatest(signOnActions.CONFIGURE_META_MASK, configureMetaMask)
 }
 
+function* watchFollowHotAndNew() {
+  while (
+    yield all([
```

Comment:
I think I would prefer that we update the `followArtists` saga and use the existing flow there rather than having a different paradigm for playlist auto follows

I also this should be more generic -- some sort of `followCollections` saga that can be agnostic to the specific playlist/album it is (like how we do artists)